### ISSUE TITLE
Link not pointing to correct URL

### ISSUE THREAD

**Issue by mehmetadali** [backdrop-contrib/og#149]
Bug in OG

If create a site within hosting solution with a path

eg.
http://mydomain.com/testsite

Install OG, create  Group,
then click on link in Content Types,Organic Groups, 'Manage Fields'

Link takes you to 
http://mydomain.com/admin/structure/types/manage/loi/fields

and not 
http://mydomain.com/testsite/admin/structure/types/manage/loi/fields

**Comment by argiepiano** [comment]
Hi @mehmetadali. Thanks for reporting.

What you are reporting is not a problem with Organic Groups, but rather a problem with your installation. The link `admin/structure/types/manage/CONTENT_TYPE/fields` is not created by Organic Groups, but by Backdrop's core.

**Comment by izmeez** [comment]
I agree the link `admin/structure/types/manage/CONTENT_TYPE/fields` works as expected and this issue is a site installation specific problem. The issue should probably be closed, unless @mehmetadali can elaborate on the problem.

**Comment by argiepiano** [comment]
Closing as no further activity.
